### PR TITLE
Feature/terminology

### PR DESCRIPTION
**Pull request by cmarin21**
Creating a pull request for merging feature/terminology -> develop branch.

### CODE REVIEW

File: xml/cap_admin_configuration_changes.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   the <command>helm upgrade</command> command.
  </para>
  <warning>
-  <title>Do not make changes to pod counts during a version upgrade</title>
+  <title>Do Not Make Changes to pod counts During a Version Upgrade</title>
```
Comment (btat) [inline xml/cap_admin_configuration_changes.xml]: `pod counts` should be changed too

File: xml/cap_admin_minibroker.xml [hunk 2]
```diff
@@ -22,8 +22,8 @@
   <link xlink:href="https://github.com/osbkit">Microsoft Azure team</link>. It
   provides a simple method to provision service brokers on Kubernetes clusters.
  </para>
- <important>
-  <title>Minibroker Upstream Services</title>
```
Comment (btat) [inline xml/cap_admin_minibroker.xml]: These 2 lines are part of the updated Minibroker messaging (https://github.com/SUSE/doc-cap/pull/431) and should be left in.

File: xml/cap_admin_minibroker.xml [hunk 3]
```diff
@@ -22,8 +22,8 @@
...
+ <warning>
+  <title>Do Not Use Minibroker on Production Systems</title>
```
Comment (btat) [inline xml/cap_admin_minibroker.xml]: See above comment. These 2 lines should be removed. (They're also the cause of the Travis builds failing)

**Comment by btat** [comment]
The PR looks great :+1: , only a couple minor fixes then I will merge it.

File: xml/app_scf_values_yaml.xml [hunk 4]
```diff
@@ -2085,7 +2085,7 @@ sizing:
   #
   # - uaa-create-user: Create the initial user in UAA
   #
-  # - configure-scf: Uses the cf CLI to configure SCF once it's online (things
+  # - configure-scf: Uses the cf CLI to configure SCF after it's online (things
```
Comment (fsundermeyer) [inline xml/app_scf_values_yaml.xml]:
```suggestion
  # - configure-scf: Uses the cf CLI to configure SCF after it is online (things
```

File: xml/cap_admin_minibroker.xml (hunk 3 above)
Comment (cmarin21) [reply xml/cap_admin_minibroker.xml]: I do not understand what should I change exactly. We can talk 1-1 to understand it better.
What I get so far is that I should remove the lines marked with red, and keep the ones marked with green.

Comment (fsundermeyer) [reply xml/cap_admin_minibroker.xml]: You changed the code to this:

```
 <warning>
  <title>Do Not Use Minibroker on Production Systems</title>
  <para>
   The services deployed by Minibroker are sourced from the stable upstream
   charts repository, see
   <link xlink:href="https://github.com/helm/charts/tree/master/stable"/>,
   and maintained by contributors to the Helm project. Though &suse; supports
   Minibroker itself, it does not support the service charts it deploys.
   Operators should inspect the charts and images exposed by the service plans
   before deciding to use them in a production environment.
  </para>
 </important>
```

and are causing a validation error. Either use <warning></warning> or <important></important>, but not a mixture of both.

**Pull request merged**